src/browser/api/system.js

```diff
@@ -700,6 +701,10 @@ exports.System = {
         }
     },
 
+    getPluginModules: function(cb) {
+        return plugins.getModules().then(cb);
```

Comment:
Does this need a `.catch()` to prevent the case where an error keeps the callback / ack waiting for ever?

Reply:
None of the functions it goes through fail, so should be okay

Reply:
:smile: none are _designed_ to fail